include/zephyr/toolchain/gcc.h

```diff
@@ -643,6 +643,12 @@ do {                                                                    \
 #define __noasan /**/
 #endif
 
+#if defined(CONFIG_UBSAN)
+#define __noubsan __attribute__((no_sanitize("undefined")))
+#else
+#define __noubsan /**/
```

Comment:
what's the `/**/` for? some odd compiler compatibility?

Reply:
(Just copypasting + blindness)